File: snippets/proxy/send-message/send_message.java
package proxy;

import com.twilio.Twilio;
import com.twilio.rest.proxy.v1.service.session.participant.MessageInteraction;

public class send_message {

    private static final String ACCOUNT_SID = System.getenv("ACCOUNT_SID");
    private static final String AUTH_TOKEN = System.getenv("AUTH_TOKEN");

    public static void main(String[] args) {
        Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

        MessageInteraction messageInteraction = MessageInteraction.creator(
                System.getenv("PROXY_SERVICE_SID"),
                "KCxxxxxxxxxxxxxxxxxxxxxxxxxx",
                "KPxxxxxxxxxxxxxxxxxxxxxxxxx",
                "Reply to this message to chat!")
            .create();

        System.out.println(messageInteraction.getSid());

        System.out.println("+ Proxy service session, message interaction SID: " + messageInteraction.getSid());
    }
}

File: snippets/proxy/send-message/send-message.py
# Download the helper library from https://www.twilio.com/docs/python/install
from twilio.rest import Client

# Your Account SID and Auth Token from twilio.com/console
account_sid = 'ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
auth_token = 'xxxxxxxxxxxxxxxxxxxxxxxxxxx'
client = Client(account_sid, auth_token)

proxy_sid = 'KSxxxxxxxxxxxxxxxxxxxxxxxxxx'
session_sid = 'KCxxxxxxxxxxxxxxxxxxxxxxxxxx'
participant_sid = 'KPxxxxxxxxxxxxxxxxxxxxxxxxx'

message_interaction = client.proxy \
                   .services(proxy_sid) \
                   .sessions(session_sid) \
                   .participants(participant_sid) \
                   .message_interactions \
                   .create(body='Reply to this message to chat!')

print(message_interaction.sid)
